### ISSUE TITLE
SVE - Q5 - Class Signal 4

### ISSUE THREAD

**Issue by PBYetml** [PBYetml/Question_ReponsesEtudiants#131]
https://github.com/PBYetml/Question_ReponsesEtudiants/blob/57da487ff4de0925c10134ee925697b51d058cce/SVE/Q5/Fichier/Signal.h#L18

pourquoi avoir besoin d'implémenter le mot clé **`virtual`**. à quoi sert-il ? est-il utilisé au bon endroit ?

**Comment by santivaliante** [comment]
Non, cela est pour la fonction CalculerEchantillon(int Amplitude, int DutyCycle). Cela sert à utilisé la fonction dans la Classe enfant (Triangle).

**Comment by PBYetml** [comment]
OKI - sur cette instruction (prototype) que devez-vous modifier ?
